app/helpers/container_helper/textual_summary.rb

```diff
@@ -35,4 +36,40 @@ def textual_restart_count
   def textual_backing_ref
     {:label => "Backing Ref (Container ID)", :value => @record.backing_ref}
   end
+
+  def textual_capabilities_add
```

Comment:
@zakiva return all these ones only if they're holding a value, e.g.

    def textual_capabilities_add
      {:label => "Add Capabilities", :value => @record.container_definition.capabilities_add} if @record.container_definition.capabilities_add
    end

Then update the screenshot as well.